app/plugins/adp-backend/seedData/delivery_programme.js

```diff
@@ -12,6 +20,15 @@ exports.seed = async function seed(knex) {
     if (parseInt(count) === 0) {
       await knex(TABLE_NAME).insert(await resolveReferences(record));
     }
+    const recordsToUpdate = recordsWithEmptyFields.filter(d => d.name === record.name)
+    if(recordsToUpdate.length > 0) {
+      await knex(TABLE_NAME)
+      .whereIn('name', recordsToUpdate)
```

Comment:
There will only ever be 1 record with any given name, so we can use `.where('name', record.name)`

Reply:
Just to clarify, recordsToUpdate will include the existing records with empty fields, if I do record.name wouldn't it change the fields even if the existing record has the description/delivery programme code populated?